src/meta/name.rs

```diff
@@ -14,108 +15,96 @@ pub enum DisplayOption<'a> {
 
 #[derive(Clone, Debug, Eq)]
 pub struct Name {
-    pub name: String,
+    name: Option<String>,
```

Comment:
Probably drop a comment on the use of `name` here. Might get confusion later otherwise.

Reply:
From what I understand `name` is only used for `.` and `..`. I think we should probably mention it here.